src/routes/transactions/helpers/swap-order.helper.ts

```diff
@@ -16,6 +16,9 @@ import { SwapsRepositoryModule } from '@/domain/swaps/swaps-repository.module';
 
 @Injectable()
 export class SwapOrderHelper {
+  private readonly restrictApps =
+    this.configurationService.getOrThrow('swaps.restrictApps');
```

Comment:
Nit: `restrictApps` is of type `unknown`, I'd declare it as boolean:

```
private readonly restrictApps: boolean = ...
```

Or parametrise the function call:

```
this.configurationService.getOrThrow<boolean>('swaps.restrictApps')
```